packages/contracts/contracts/optimistic-ethereum/OVM/bridge/tokens/OVM_L1ERC20Gateway.sol

```diff
@@ -5,7 +5,12 @@ pragma experimental ABIEncoderV2;
 
 /* Interface Imports */
 import { iOVM_L1TokenGateway } from "../../../iOVM/bridge/tokens/iOVM_L1TokenGateway.sol";
-import { Abs_L1TokenGateway } from "./Abs_L1TokenGateway.sol";
+import { iOVM_L2DepositedToken } from "../../../iOVM/bridge/tokens/iOVM_L2DepositedToken.sol";
+
+/* Library Imports */
+import { OVM_CrossDomainEnabled } from "../../../libraries/bridge/OVM_CrossDomainEnabled.sol";
+import { UniSafeMath } from "../../../libraries/standards/UniSafeMath.sol";
```

Comment:
Just curious why you chose this over OZ safe math?

Reply:
We seem to be using both, I wasn't sure. I've switched to OZ version now.